### PR TITLE
Prevent the user from launching multiple jobs by rapidly clicking on buttons

### PR DESCRIPTION
**Pull request by mabashian**
##### SUMMARY
link https://github.com/ansible/awx/issues/14187

The easiest way to address this was to go right to the source of the launch/relaunch requests.  We were already keeping track of a launching "state" so I hooked into that at the beginning of each function.  If we detect that we're already launching then we do nothing.

The launch buttons already have logic that disables them while the launch request is firing.  Disabling the launch button inside the prompt wizard (very last step) is much more difficult and would probably require some refactoring.  I decided not to go down that route and figured I'd be more likely to introduce new bugs if I did.

##### ISSUE TYPE
 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME
 - UI

##### AWX VERSION
22.5.0 and earlier

### CODE REVIEW

File: awx/ui/src/components/LaunchButton/LaunchButton.js
Comment (vidyanambiar) [inline awx/ui/src/components/LaunchButton/LaunchButton.js:63]: Could we add a toast message here to indicate that the job was launched?

**Comment by mabashian** [comment]
Here's the toast in action:
template_launch_toast

**Review by vidyanambiar — APPROVED** [review]
lgtm

**Review by akus062381 — COMMENTED** [review]
I didn't get to submit my approval before merge, but I did test this.

**Pull request merged**
